server/data/cache/PrisonRegisterStore.ts

```diff
@@ -1,5 +1,5 @@
 import redis from 'redis'
-import type { Prison } from 'prisonTypes'
+import type { PrisonDto } from 'prisonRegisterApiClient'
```

Comment:
I've changed the `PrisonRegisterStore` to cache `PrisonDto`'s rather than our simpler `Prison` type as before. This is because the `PrisonDto` contains the address (`AddressDto`) which we will need later (not in this PR, but later in this story)